### PR TITLE
Add is_nested=False

### PR DESCRIPTION
**Pull request by Fuzzkatt**
Attempt to fix https://github.com/Lightning-AI/lightning-thunder/issues/287

### CODE REVIEW

**Comment by Fuzzkatt** [comment]
cc @mruberry: Following your suggestion https://github.com/Lightning-AI/lightning-thunder/issues/93#issuecomment-2030416883 that you linked in the original issue, I found out how to do the first half of the suggestion setting is_nested to always False. Could you review and add some suggestions on how to add the second half (ensuring that programs using nestedtensors always fail)?

File: thunder/torch/__init__.py [hunk 1]
```diff
@@ -184,6 +184,13 @@ def is_cuda(a: TensorLike, /) -> bool:
     return a.device.devicetype is devices.DeviceType.CUDA
 
 
+# is nested always returns False for now:
+# https://github.com/Lightning-AI/lightning-thunder/issues/93#issuecomment-2030416883
+@torchsymbol(torch.Tensor.is_nested, is_property=True, id="torch.is_nested")
```
Comment (mruberry) [inline thunder/torch/__init__.py:189]: fyi @k223kim, thanks to your work it's easy to add more properties!

File: thunder/torch/__init__.py [hunk 2]
```diff
@@ -184,6 +184,13 @@ def is_cuda(a: TensorLike, /) -> bool:
     return a.device.devicetype is devices.DeviceType.CUDA
 
 
+# is nested always returns False for now:
```
Comment (mruberry) [inline thunder/torch/__init__.py:187]: Nice comment

**Comment by mruberry** [comment]
This is great, @Fuzzkatt, and I think we can stop this PR here. It does exactly what it should. Failing when provided with a nested tensor can be a separate issue (and should maybe be checked in the prologue, fyi @t-vi). 

Would you add a test for this property, following the pattern here:

https://github.com/Lightning-AI/lightning-thunder/blob/843b2c265ec335aa1406f3d5650324567b88bf78/thunder/tests/opinfos.py#L548

and then let's merge this!

File: thunder/torch/__init__.py (hunk 1 above)
Comment (k223kim) [reply thunder/torch/__init__.py:189]: Happy to help! 🚀

**Review by mruberry — APPROVED** [review]
Cool! Let's see if this unblocks operators that were testing for `is_nested`.

@Borda and @t-vi to merge

**Comment by mruberry** [comment]
@t-vi this needs to be merged manually because of flaky ci

**Comment by t-vi** [comment]
@Fuzzkatt @mruberry  Thank you. Could you please resolve the merge conflicts? I'll check for CI issues and override as eneded.

**Pull request merged**